packages/shared/src/components/referral/SearchReferralButton.tsx

```diff
@@ -40,14 +39,16 @@ export function SearchReferralButton({
   }
 
   return (
-    <Button
+    <button
       type="button"
-      variant={ButtonVariant.Float}
-      className={classNames('bg-theme-overlay-from', className)}
+      className={classNames(
+        'flex h-10 flex-row items-center rounded-12 bg-theme-overlay-from px-3 font-bold text-theme-label-tertiary typo-callout hover:text-theme-label-primary',
+        className,
+      )}
       onClick={handleClick}
     >
       {availableCount}
       <KeyReferralIcon size={IconSize.Medium} className="-mr-1 ml-1 mt-1" />
-    </Button>
+    </button>
```

Comment:
I would honestly prefer if we force the margins instead. This way if we change the button styling in the future, this will need to be adjusted as well. And I doubt we will remember, until we see again that it looks different...

Reply:
Do you mean the padding? Margins left and right look to be okay. I agree with your point, though I think this is a special case where we only have one of this kind of button. So I guess, we can move forward with this for now.

Reply:
Yup, sorry, padding is what I meant. Next time I'll say spacing :D

Reply:
And as I said above, feel free to merge this. It's just my preference for future-proofness. Which I'm not sure if it's that big of a deal, since it's just visual change.